server/server.ts

```diff
@@ -98,6 +99,22 @@ io.on("connection", (socket) => {
     io.to(host.socketId).emit("GAME_CREATED", gameCode);
   });
 
+  socket.on("START_GAME", (gameCode) => {
+    console.log(`Game Started: ${gameCode}`);
+
+    const game: Game | undefined = gamesMap.get(gameCode);
+    const players = game?.getPlayers();
+
+    if (game == undefined) {
+      console.log(`Game : ${gameCode} does not exist`);
+      return;
+    } else if (players != undefined) {
```

Comment:
Might want to change this check to check for atleast 1 (maybe two players). This could also potentially be achieved on the frontend, but not a high priotity.